### PR TITLE
Nick #34

### PR DESCRIPTION
**Pull request by Megatron4537**
The TA can choose a group size and submit, teams will be made automatically and he can also undo all teams for his class if he wishes to change teams' size. DB makes according changes.

### CODE REVIEW

**Comment by Megatron4537** [comment]
this can be tested on www.users.encs.concordia.ca/~n_hortaa
login as mike@gmail.com to see

**Comment by Megatron4537** [comment]
If you hit refresh upon team creation, deletion, team id will be displayed or not, @TheAlexRobert , @SaiShan can you do front-end for this and possibly make it so that if teams are made, display tables for each instead of just big list like it is right now.

**Comment by SaiShan** [comment]
works for me, tested it, nice job

And yes, will do, we were waiting for the merge to work on the front end!

**Review by SaiShan — APPROVED** [review]

File: code/php/inSession/myProfile.php [hunk 1]
```diff
@@ -68,39 +77,141 @@
...
+			}
+			//upload apge
+			echo "<a href='viewGroup.php'>
+			   <input type='button' value='upload'class='button' />
+			</a>";
+
 			?>
-		
-</div>		
+
```
Comment (Davidster) [inline code/php/inSession/myProfile.php]: Could you fix up the indentation of this section?

**Pull request merged**